CMakeLists.txt

```diff
@@ -230,6 +232,12 @@ if (ENABLE_EXTERNAL_ROCPRIM)
     rocPRIM)
 endif ()
 
+if (ENABLE_SYCL)
+  set (raja_depends
+    ${raja_depends}
+    sycl)
```

Comment:
Where does this sycl target come from? Or is this just used so that CMake will generate `-lsycl`?

Reply:
It looks like this is unnecessary currently.  All that is needed to build is the `-fsycl` and `-fsycl-unnamed-lambda` compiler flags.  It wasn't clear if there would be support in cmake or blt when it was added.